deployment/armada-bundle/templates/pulsar-init-job.yaml

```diff
@@ -0,0 +1,43 @@
+{{- if .Values.pulsar.init.enabled }}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: pulsar-init
+spec:
+  template:
+    spec:
+      initContainers:
+        - name: wait
+          image: alpine:3.16
+          command:
+            - /bin/sh
+            - -c
+            - |
+              echo "Waiting for Pulsar... ({{ .Values.pulsar.init.brokerHost }}:{{ .Values.pulsar.init.port }})"
+
+              while ! nc -z {{ .Values.pulsar.init.brokerHost }} {{ .Values.pulsar.init.port }}; do
+                sleep 1
+              done
+
+              echo "Pulsar started!"
+      containers:
+        - name: init
+          image: apachepulsar/pulsar:2.10.2
```

Comment:
Could we make this a template since mac has a different image?

Reply:
makes sense